src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php

```diff
@@ -41,25 +41,16 @@ abstract class AbstractSessionListener implements EventSubscriberInterface, Rese
     public const NO_AUTO_CACHE_CONTROL_HEADER = 'Symfony-Session-NoAutoCacheControl';
 
     /**
+     * @param array<string, mixed> $sessionOptions
+     *
      * @internal
      */
-    protected ?ContainerInterface $container;
-
-    private bool $debug;
-
-    /**
-     * @var array<string, mixed>
-     */
-    private array $sessionOptions;
-
-    /**
-     * @internal
-     */
-    public function __construct(?ContainerInterface $container = null, bool $debug = false, array $sessionOptions = [])
-    {
-        $this->container = $container;
-        $this->debug = $debug;
-        $this->sessionOptions = $sessionOptions;
+    public function __construct(
+        /** @internal */
```

Comment:
What is the meaning of this `@internal` annotation here? It was on the method.

Reply:
The `$container` property was flagged `@internal` previously and I don't want to lose that.

Reply:
let's make it private then, and add the same private property on child classes?

Reply:
Good idea.